_tests/tests/__snapshots__/E2eTest__test_e2e_no_op__1.php

```diff
@@ -68,27 +68,27 @@
                             "Store owner can complete onboarding wizard": [
                                 {
                                     "title": "can complete the \\"Store Details\\" section",
-                                    "status": "passed"
+                                    "status": "pending"
                                 },
                                 {
                                     "title": "can complete the industry section",
-                                    "status": "passed"
+                                    "status": "pending"
                                 },
                                 {
                                     "title": "can save industry changes when navigating back to \\"Store Details\\"",
-                                    "status": "passed"
+                                    "status": "pending"
                                 },
                                 {
                                     "title": "can discard industry changes when navigating back to \\"Store Details\\"",
-                                    "status": "passed"
+                                    "status": "pending"
                                 },
                                 {
                                     "title": "can complete the product types section",
-                                    "status": "passed"
+                                    "status": "pending"
                                 },
                                 {
                                     "title": "can complete the business section",
-                                    "status": "passed"
+                                    "status": "pending"
```

Comment:
Tests are passing, the number of successes and pending are fine, but I don't know why these are being skipped now, maybe we could double-check with Solaris

Reply:
Hey @Luc45 -- some of the onboarding tests have been marked "skip" for a while now because development was going pretty fast with the onboarding wizard (I think it's called "core profiler" now).  Instead of constantly chasing failures, we just opted to skip some of the OBW tests for now.

Reply:
Thanks for the context @lanej0! That's super helpful to know!